scripts/support/create_ge_images/README.rst

```diff
@@ -22,8 +22,8 @@ The software must be used from a Ubuntu 14.04 system with the following packages
 * libguestfs-tools
 
 The scripts should be copied to */opt/create_ge_images* to share the installation
-between several users. Also the module *osclients.py*, provided in the parent
-folder, must be copied.
+between several users. Also the module *osclients.py*, provided in the utils
```

Comment:
Ok, here is a little different, because the installation instructions are for installing only the required files in a specific path.

Anyway, I've changed import osclients -> import utils/osclients and in the instructions put that the utils directory of the root project must be also coped.